packages/grid/enclave/attestation/server/cpu_attestation.py

```diff
@@ -14,7 +14,16 @@ def attest_cpu() -> str:
     logger.debug(f"Stderr: {cpu_report.stderr}")
 
     logger.info("Attestation Return Code: {}", cpu_report.returncode)
+    res = "False"
     if cpu_report.returncode == 0 and cpu_report.stdout == "true":
-        return "True"
+        res = "True"
 
-    return "False"
+    # Fetch token from Micrsoft Attestation library
+    cpu_token = subprocess.run(
```

Comment:
Leaving a note , for later, we do two calls, to the attestation client library , we could combine them in to a single call in later PR's

Reply:
Will shelf this as a TODO item, good point though.